pages/favs.jsx

```diff
@@ -3,18 +3,64 @@ import { getFavs } from '../js/favs-manager';
 import ProjectList from '../components/project-list/project-list.jsx';
 import HintMessage from '../components/hint-message/hint-message.jsx';
 
+import Service from '../js/service.js';
+import config from '../config/config.js';
+import env from "../config/env.generated.json";
+
 export default React.createClass({
   getInitialState() {
     return {
-      favs: null
+      favs: null,
+      loggedInCheckDateTime: null
     };
   },
+  verifyLoggedInStatus() {
+    Service.nonce()
+      .then((response) => {
```

Comment:
no need for parens in an arrow function if there's only one argument:
```
...
.then(response => this.updateLoggedInInfo(true, response.user))
...
```

Also no need for `{ return ... }` decoration if an arrow function is a one-liner, which is quite nice.